src/adapter/z-stack/znp/znp.ts

```diff
@@ -308,6 +312,30 @@ class Znp extends events.EventEmitter {
         });
     }
 
+    public requestZdo(clusterId: ZdoClusterId, payload: Buffer, waiterID?: number): Promise<void> {
```

Comment:
My expectation was that we don't use the ZNP specific ZDO commands but just `AF_DATA_REQUEST`, wouldn't it be easier to use that?

Reply:
Don't know if the ZDO rsp payloads will match the same?
Probably have to rework the req payloads (looks like the network address would no longer be prefixing at least?).
Does it work for `NETWORK_ADDRESS_REQUEST` too (just wondering since it has to use the ieee from the zdo payload and not dstaddr from data req)?
Something like this would work out-of-the-box?
```typescript
// await this.znp.requestZdo(clusterId, payload, waiter?.ID);
await this.dataRequest(networkAddress, Zdo.ZDO_ENDPOINT, Zdo.ZDO_ENDPOINT, clusterId, Constants.AF.DEFAULT_RADIUS, payload, 15000);
```

Responses would still come in as AREQ/ZDO?

Reply:
As discussed, let's keep doing the native AREQ requests as other things might break once we start to use `DATA_REQUEST`